### PR TITLE
Redirect defunct /gallery to /projects

### PR DESCRIPTION
**Pull request by Erin007**
Similar to #33616 

Years ago we made a project gallery for Artist and PlayLab projects at studio.code.org/gallery:

Screen Shot 2020-03-12 at 8 36 53 PM

We no longer support this page in favor of studio.code.org/projects.  As part of long overdue process to completely deprecate the GalleryActivities that generate that old gallery, in this PR I'm redirecting /gallery to /projects. 

gallery-redirect

### CODE REVIEW

File: dashboard/config/locales/en.yml [hunk 1]
```diff
@@ -696,20 +692,6 @@ en:
       body: "Students can explore concepts from our Computer Science Principles course hands-on using these digital tools. Use the widgets on their own, or create a single-concept lesson for your class."
       audience: "Ages 14+"
       cta: "Learn how"
-  gallery:
```
Comment (Erin007) [inline dashboard/config/locales/en.yml:699]: Deleting these gallery strings is the relevant change here.  Everything else is just an auto change of 'single-quotes' to "double-quotes"

File: dashboard/config/routes.rb [hunk 2]
```diff
@@ -27,12 +27,10 @@
...
 
-  resources :gallery_activities, path: '/gallery' do
-    collection do
-      get 'art', to: 'gallery_activities#index', app: Game::ARTIST
-      get 'apps', to: 'gallery_activities#index', app: Game::PLAYLAB
-    end
-  end
+  # Redirects to /projects
+  # TODO: (ErinB) remove when gallery_activities are fully deleted
+  get "/gallery", to: "gallery_activities#index"
```
Comment (maddiedierker) [inline dashboard/config/routes.rb]: we can redirect directly from the router ([rails source](https://guides.rubyonrails.org/routing.html#redirection)) like this:

```
get '/gallery', to: redirect('/projects')
```

then we can totally remove the `gallery_activities#index` method

**Review by maddiedierker — APPROVED** [review]
left a little extra cleanup comment we can do, but not a merge-blocker. 

thank you for this!!! 🎉 i didn't even know this page existed 😬

**Comment by Erin007** [comment]
rolling into #33616

**Pull request closed without merging**